build.gradle

```diff
@@ -113,24 +113,17 @@ bintray {
 }
 
 dependencies {
-    api platform('com.exactpro.th2:bom:2.7.+')
-    implementation "com.exactpro.th2:common"
-    implementation 'com.exactpro.th2:sailfish-utils'
-    implementation "com.exactpro.th2:grpc-common"
-    implementation "com.exactpro.th2:grpc-check1"
+    api platform('com.exactpro.th2:bom:2.10.1')
+    implementation 'com.exactpro.th2:common:2.9.0'
+    implementation 'com.exactpro.th2:sailfish-utils:2.2.2'
+    implementation 'com.exactpro.th2:grpc-common:2.3.2'
+    implementation 'com.exactpro.th2:grpc-check1:2.2.1'
     implementation "org.slf4j:slf4j-log4j12"
     implementation "org.slf4j:slf4j-api"
 
-    implementation "com.google.protobuf:protobuf-java-util"
-    implementation "io.grpc:grpc-protobuf"
-    implementation "io.grpc:grpc-stub"
-    implementation "io.grpc:grpc-core"
-    implementation "io.grpc:grpc-netty"
-
     implementation "com.exactpro.sf:sailfish-core:${sailfishVersion}"
```

Comment:
Please, check the Sailfish version and replace it with a snapshot if it is a release version. Other boxes use snapshot version so I think we should use it here as well